docs/kubernetes/kctlr-manage-bigip-objects.rst

```diff
@@ -320,39 +207,39 @@ If you want to delete a front-end BIG-IP virtual server, but keep its associated
             "virtualServer": {
               "backend": {
                 ...
-                }]
+                }
               },
               "frontend": {
                 "virtualAddress": {
-                  "port": 80,
-                  \\ REMOVE THE LINE BELOW
-                  "bindAddr": "1.2.3.4"
+                  "port": 80
                 },
-                "partition": "kubernetes",
+                "partition": "<partition-name>",
                 "balance": "round-robin",
                 "mode": "http"
               }
             }
           }
       kind: ConfigMap
       metadata:
-        creationTimestamp: 2017-02-14T17:24:34Z
+        creationTimestamp: 2018-02-14T17:24:34Z
```

Comment:
This is a field created by the API server. It's programmatically generated metadata and we don't need it in the example. Also a hassle if we'll be updating the field.

Reply:
I took this example out, I don't think it really added anything of value.